package.json

```diff
@@ -74,5 +74,8 @@
     "tslint": "~5.11.0",
     "typescript": "~2.9.2",
     "zone.js": "~0.8.26"
+  },
+  "peerDependencies": {
+    "@angular/common": "~5.0,~6.0"
```

Comment:
Doesn't `~6.0` mean `>= 6.0.0 && < 6.1.0` (see https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1)?
I think, this would exclude the current angular release `6.1.x`.

Reply:
You are right! I have always hated this counter-intuitive special symbols…

Reply:
Fixed in dc577e8a